lib/src/utils.ts

```diff
@@ -34,6 +34,22 @@ const prefixStringStartingWithNumberIfNeeded = (str: string) => {
 const pathParamWithBracketsRegex = /({\w+})/g;
 const wordPrecededByNonWordCharacter = /[^\w\-]+/g;
 
+export const pathParamToVariableName = (name: string) => snakeToCamel(name.replaceAll("-", "_"));
+
+export const replaceHyphatedPath = (path: string) => {
+    const matcherRegex = /{(\b\w+(?:-\w+)*\b)}/g;
```

Comment:
could you just move this one line up, otherwise it would be re-created on each call of `replaceHyphatedPath`